### PR TITLE
testing/async-profiler: new aport

### PR DESCRIPTION
**Pull request by jchipmunk**
https://github.com/jvm-profiling-tools/async-profiler
Sampling CPU and HEAP profiler for Java featuring AsyncGetCallTrace + perf_events

### CODE REVIEW

**Comment by jchipmunk** [comment]
@nmeum, could you review this PR?

**Comment by jchipmunk** [comment]
@andypost, could you pay attention to this MR?

**Comment by andypost** [comment]
@jchipmunk travis still fails, please squash commits to one with new aport

**Comment by andypost** [comment]
It looks good to go, btw is there upstream PR filed for patch?

**Comment by jchipmunk** [comment]
jvm-profiling-tools/async-profiler#196

File: testing/async-profiler/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	sudo bash -c 'echo 1 > /proc/sys/kernel/perf_event_paranoid'
```
Comment (andypost) [inline testing/async-profiler/APKBUILD]: no idea how builders will work, needs someone more familiar with it /cc @ncopa

Comment (jchipmunk) [reply testing/async-profiler/APKBUILD]: This MR is still relevant.

Comment (Ikke) [reply testing/async-profiler/APKBUILD]: Not sure if this is acceptable, but if it would, you could just use `sh -c`, instead of `bash -c` here.

Comment (3ju) [reply testing/async-profiler/APKBUILD]: I surely hope that the builders do not allow sudo without authentication.

Comment (jchipmunk) [reply testing/async-profiler/APKBUILD]: I excluded tests depended to modify `/proc/sys/kernel/perf_event_paranoid`.

**Pull request closed without merging**